.changeset/eight-nails-accept.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/perseus": minor
+"@khanacademy/perseus-editor": minor
```

Comment:
Our version numbers don't mean much, but I would consider this very much a major change.